packages/hash/frontend/src/blocks/page/createSuggester/BlockSuggester.tsx

```diff
@@ -14,22 +14,18 @@ export interface BlockSuggesterProps {
   sx?: SxProps<Theme>;
 }
 
-// @todo remove this when API returns actual icon URL
 export const getVariantIcon = (option: {
   variant: BlockVariant;
   meta: RemoteBlockMetadata;
 }): string | undefined => {
-  if (option.variant.icon?.startsWith("/")) {
-    return `https://blockprotocol.org${option.variant.icon}`;
-  }
+  const iconPath = option.variant.icon;
```

Comment:
`BlockVariant` typings are probably misleading. I guess we can imagine that `BlockVariant.icon` is `undefined` unless it is strictly against the BP. Our past code handles `undefined`, but after the change we'll get an exception here: `iconPath.replace(/^\//, "")`.